### DISCUSSION TITLE
Timeline

### DISCUSSION THREAD

**Discussion by rbureau85** [RocketCommunicationsInc/astro#15] [Q&A]
Will the timeline be in the new stencil format? I really need it for my project. The old one barely works and I had to finagle your source code to get it to run in react

**Comment by markacianfrani** [comment]
@rbureau85 we've decided to deprecate the Timeline component for now. we found that the current implementation has quite a few issues and we intend to rewrite it from scratch in the new stencil repo. I don't have a clear timeframe as to when yet, but I will keep this discussion updated once I do. not a great answer I know, but timeline is probably the most complex component so there's a lot for us to consider

**Comment by rbureau85** [comment]
Mark,
Thank  you for the update. If it helps speed it up we have a mission need
for it at SSC. We are standardizing all of our missions to use the Astro
platform and one of our current taskings require we have a timeline and we
would like to utilize the Astro component as that is the SMC standard for
platform design :-)

**Reply by dmcalester** [reply]
Hi @rbureau85 would you mind reaching out to via (duncan@rocketcom.com) I’d like to understand your needs and timeframes to see how what options we have on our end to support you and the SSC/SMC mission

**Comment by rbureau85** [comment]
Yes, of course. Sending you an e-mail from my work accounts.
